code/headers/HC_SR04_c.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include <hwlib.hpp>
+
+namespace R2D2::Distance
+{
+        /**
+     *  \brief
+     *  This class is for reading the data from the HC-SR04 ultrasone sensor
+     * 
+     *  \details
+     *   This class return the distance measured with the HC-SR04 ultrasone sensor, accurate up to 400 centimeters.
+     *   After 400 centimeters measurements are unuseable and should be disgarded.
+     */
+    class HC_SR04_c
```

Comment:
A bit strange that the *HC_SR04_c* class is added in a branch called *Feature game c* that also adds a *game_c* class.

Reply:
I know, stranger things are happening